c2corg_ui/static/partials/suggestions/articles.html

```diff
@@ -0,0 +1,6 @@
+  <span class="suggestion-text"> 
+    <span class="suggestion-title">
+      <span ng-bind-html="highlight(label, _query)"></span> 
+    </span><br>
+  </span>
+  <span class="icon-container"><span ng-bind-html="::activitiesHtml"></span></span>
```

Comment:
> I saw some icons in the other cases so I tried to add this also. And in case of "routes" there are also same icons. This is the case how it looks with many activities...

OK. Let's do it like this. Testers will tell us if this is a problem.

> Problem is sometimes that long title - then the icons are pushed under each other instead of being next to each other.

I think we have the same problem with other document types (for instance you may have routes with very long title). In that case, the title should be wrap to a second line.
Could you create an issue for this?